src/main/web_src/src/components/fagsystem/sykdom/form/partials/DetaljertSykemelding.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react'
+import { FormikDatepicker } from '~/components/ui/form/inputs/datepicker/Datepicker'
+import { FormikTextInput } from '~/components/ui/form/inputs/textInput/TextInput'
+import { FormikCheckbox } from '~/components/ui/form/inputs/checbox/Checkbox'
+import { FormikSelect } from '~/components/ui/form/inputs/select/Select'
+import { FormikDollyFieldArray } from '~/components/ui/form/fieldArray/DollyFieldArray'
+import { Kategori } from '~/components/ui/form/kategori/Kategori'
+import { SelectOptionsManager as Options } from '~/service/SelectOptions'
+import { OrganisasjonMedArbeidsforholdSelect } from '~/components/organisasjonSelect'
+import { SelectOptionsDiagnoser } from './SelectOptionsDiagnoser'
+import LegeSelect from './LegeSelect'
+import { ArbeidKodeverk } from '~/config/kodeverk'
+import { SykemeldingForm, Lege, Arbeidsgiver } from '~/components/fagsystem/sykdom/SykemeldingTypes'
+
+type DiagnoseSelect = {
+	diagnoseNavn: string
+}
+
+const initialValuesDiagnose = {
+	diagnose: '',
+	diagnosekode: '',
+	system: ''
+}
+
+const initialValuesPeriode = {
+	aktivitet: {
+		aktivitet: null as string,
+		behandlingsdager: 0,
+		grad: 0,
+		reisetilskudd: false
+	},
+	fom: '',
+	tom: ''
+}
+
+export const DetaljertSykemelding = ({ formikBag }: SykemeldingForm) => {
+	const diagnoseSystem = '2.16.578.1.12.4.1.1.7170'
```

Comment:
Kunne lagt denne utenfor metoden og skrive med store bokstaver for å ilustere at det er en const. 

diagnoseSystem er vel feil navn det egentlig er T***eller annet som står i sykemeldingen.